### ISSUE TITLE
Few more commands for private-chats

### ISSUE THREAD

**Issue by balavishnuvj** [kentcdodds/kcd-discord-bot-v1#24]
Problem description: To improve and add more capability to `?private-chat`

Suggested solution:
Currently, we have only on command. Suggestions for few more.

- [ ]  Create `?private-chat create @user1 @user2` (can from any channel)
- [ ]  Add new user `?private-chat add @user1 @user2` 
- [ ]  Remove user `?private-chat remove @user1 @user2`
- [ ]  Delete `?private-chat delete-channel`
- [ ]  Extend `?private-chat extend xminutes`

Apart from first command rest has to be executed from a private channel

**Comment by marcosvega91** [comment]
I think all commands seams interesting to me. Great !!! 💯

**Comment by balavishnuvj** [comment]
@marcosvega91 , for extend to work, we might have to store that information somewhere.... gist?

**Comment by marcosvega91** [comment]
Yes, unfortunately discord does not have any field to add expiration date on channels. 

If you want to try implement something I can suggest you to leave the extend command as last. Thanks for all @balavishnuvj 😄

**Comment by kentcdodds** [comment]
We could put it in the channel description

**Comment by marcosvega91** [comment]
ah, yeah it makes sense. Good observation !! 💯

**Comment by smeijer** [comment]
To make the commands more familiar (/similar to other programs), I would like to propose aliases:

```
?private-chat add             | alias: invite
?private-chat remove          | alias: kick
?private-chat delete-channel  | alias: close, end
```

But maybe we can also make the `invite` and `kick` top level commands to they are effective everywhere? If that's possible, it might require a separate issue:

```
?kick @user
?invite @user
```

**Comment by kentcdodds** [comment]
`?kick` and `?invite` may be easily confused with "kick from the server" and "invite to the server" so I think even though it's more verbose to type out the whole thing, we shouldn't go with top-level commands. I'm fine with aliases though.

**Comment by kentcdodds** [comment]
One other idea I had about this. What if we added a rule to the private chat cleanup: "The channel will not be removed until all participants have sent a reaction/message or it's been 4 days since the last message."

Once all participants have participated, then we can go ahead and apply the existing rules.
